app/src/main/res/layout/item_task.xml

```diff
@@ -45,7 +45,7 @@
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:paddingEnd="@dimen/hori_medium"
-            android:text="This is the task that is due!"
+            android:text="@string/taskDue"
```

Comment:
taskDue sounds ambiguous in the sense that there's a variety of places it could be used. And with different messages. Here, it has been used only as a placeholder in the code in case of any failure during inflation. So I suggest adding 'palceholder' keyword in the string name.

Reply:
Requested changes for adding "placeholder" in string name has been done